### PR TITLE
renaming variable: from nbpml to nbl

### PR DESCRIPTION
**Pull request by btrb**

### CODE REVIEW

File: benchmarks/user/benchmark.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def default_list(ctx, param, value):
                      help='Number of grid points along each axis'),
         click.option('-s', '--spacing', default=(20., 20., 20.),
                      help='Spacing between grid sizes in meters'),
-        click.option('-n', '--nbpml', default=10,
+        click.option('-n', '--nbl', default=10,
```
Comment (mloubout) [inline benchmarks/user/benchmark.py]: change help too to what 'nbl' means

File: examples/seismic/acoustic/acoustic_example.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
                         help="Operator auto-tuning mode")
     parser.add_argument("-so", "--space_order", default=6,
                         type=int, help="Space order of the simulation")
-    parser.add_argument("--nbpml", default=40,
+    parser.add_argument("--nbl", default=40,
```
Comment (mloubout) [inline examples/seismic/acoustic/acoustic_example.py]: same

File: examples/seismic/elastic/elastic_example.py [hunk 3]
```diff
@@ -64,7 +64,7 @@ def test_elastic():
                         help="Operator auto-tuning mode")
     parser.add_argument("-so", "--space_order", default=4,
                         type=int, help="Space order of the simulation")
-    parser.add_argument("--nbpml", default=40,
+    parser.add_argument("--nbl", default=40,
```
Comment (mloubout) [inline examples/seismic/elastic/elastic_example.py]: same

File: examples/seismic/model.py [hunk 4]
```diff
@@ -473,21 +473,21 @@ def initialize_function(function, data, nbpml):
         The initialised object.
     data : ndarray
         The data array used for initialisation.
-    nbpml : int
+    nbl : int
```
Comment (mloubout) [inline examples/seismic/model.py]: same

File: examples/seismic/model.py [hunk 5]
```diff
@@ -607,7 +607,7 @@ class Model(GenericModel):
         Order of the spatial stencil discretisation.
     vp : array_like or float
         Velocity in km/s.
-    nbpml : int, optional
+    nbl : int, optional
```
Comment (mloubout) [inline examples/seismic/model.py]: same

File: examples/seismic/model.py [hunk 6]
```diff
@@ -737,7 +737,7 @@ class ModelElastic(GenericModel):
         P-wave velocity in km/s.
     vs : float or array
         S-wave velocity in km/s.
-    nbpml : int, optional
+    nbl : int, optional
```
Comment (mloubout) [inline examples/seismic/model.py]: same

File: examples/seismic/model.py [hunk 7]
```diff
@@ -801,7 +801,7 @@ class ModelViscoelastic(ModelElastic):
         S-wave velocity in km/s.
     qs : float or array
         S-wave qulaity factor (dimensionless).
-    nbpml : int, optional
+    nbl : int, optional
```
Comment (mloubout) [inline examples/seismic/model.py]: same

File: examples/seismic/tti/tti_example.py [hunk 8]
```diff
@@ -52,7 +52,7 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
                         help="Operator auto-tuning mode")
     parser.add_argument("-so", "--space_order", default=4,
                         type=int, help="Space order of the simulation")
-    parser.add_argument("--nbpml", default=40,
+    parser.add_argument("--nbl", default=40,
```
Comment (mloubout) [inline examples/seismic/tti/tti_example.py]: same

File: examples/seismic/viscoelastic/viscoelastic_example.py [hunk 9]
```diff
@@ -64,7 +64,7 @@ def test_viscoelastic():
                         help="Operator auto-tuning mode")
     parser.add_argument("-so", "--space_order", default=4,
                         type=int, help="Space order of the simulation")
-    parser.add_argument("--nbpml", default=40,
+    parser.add_argument("--nbl", default=40,
```
Comment (mloubout) [inline examples/seismic/viscoelastic/viscoelastic_example.py]: same

File: benchmarks/user/benchmark.py (hunk 1 above)
Comment (btrb) [reply benchmarks/user/benchmark.py]: waiting for reply from [here](https://github.com/opesci/devito/issues/972#issue-510485866)

Comment (FabioLuporini) [reply benchmarks/user/benchmark.py]: Number of absorbing layers

Comment (FabioLuporini) [reply benchmarks/user/benchmark.py]: or

Number of absorbing ("sponge") layers

**Comment by btrb** [comment]
sorry, i messed with rebase. so will close in favor of the branch `nbpml_to_nbl3`

**Pull request closed without merging**